### PR TITLE
026 Script for requirement [PerformAudioPassThru] Speak (<errorCode>) + UI.PerformAudioPassThru (<errorCode>)

### PR DESCRIPTION
**Pull request by maria-mih**
026_ATF_P_PerformAudioPassThru_UI_success_false_TTS_success_false.lua
All common functions used available at #703
Testing coverage available at #643 as well as https://github.com/smartdevicelink/sdl_atf_test_scripts/blob/feature/extended_policy_functionality/test_sets/PAPT_new_param_audioPassThruIcon.txt

@istoimenova Please review

### CODE REVIEW

File: test_scripts/API/PerformAudioPassThru/026_ATF_P_PerformAudioPassThru_UI_success_false_TTS_success_false.lua [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+	      self.hmiConnection:SendNotification("TTS.Started",{})
+	      
+	      local function ttsSpeakResponse()
+	        self.hmiConnection:SendError(data.id, data.method, hmi_result_code_tts[cnt_tts].result_code, "")
+	        self.hmiConnection:SendNotification("TTS.Stopped")
+	      end
+	      RUN_AFTER(ttsSpeakResponse, 1000)
+	  	end)
+
+	  EXPECT_HMICALL("UI.PerformAudioPassThru",
```
Comment (istoimenova) [inline test_scripts/API/PerformAudioPassThru/026_ATF_P_PerformAudioPassThru_UI_success_false_TTS_success_false.lua]: @maria-mih, please align rows 148-186

File: test_scripts/API/PerformAudioPassThru/026_ATF_P_PerformAudioPassThru_UI_success_false_TTS_success_false.lua [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+	      bitsPerSample = "8_BIT",
+	      audioType = "PCM",
+	      muteAudio = true,
+	      audioPassThruIcon =
+		    { 
+		      value = "icon.png",
+		      imageType = "STATIC"
+		    }
+	    })
+	  	EXPECT_HMICALL("TTS.Speak",
```
Comment (istoimenova) [inline test_scripts/API/PerformAudioPassThru/026_ATF_P_PerformAudioPassThru_UI_success_false_TTS_success_false.lua]: @maria-mih, please add one empty row.

Comment (maria-mih) [inline test_scripts/API/PerformAudioPassThru/026_ATF_P_PerformAudioPassThru_UI_success_false_TTS_success_false.lua]: @istoimenova Added

File: test_scripts/API/PerformAudioPassThru/026_ATF_P_PerformAudioPassThru_UI_success_false_TTS_success_false.lua (hunk 1 above)
Comment (maria-mih) [inline test_scripts/API/PerformAudioPassThru/026_ATF_P_PerformAudioPassThru_UI_success_false_TTS_success_false.lua]: @istoimenova Aligned again with beautifier

**Comment by maria-mih** [comment]
@istoimenova Please check if script is properly aligned now on commit f5c4441c2b8ecf0c92a1d9b1f4dd96ceff9b559a

**Pull request merged**